app/assets/stylesheets/application.css.scss

```diff
@@ -168,3 +168,21 @@ nav {
 .float-right {
   float: right;
 }
+
+.new-posts {
+  color: white;
+  background: #b26e6e;
+  padding: 15px 20px;
+  display: block;
+  text-align: center;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  font-size: 14px;
+  position: fixed;
+  min-width:100%;
+  margin-top: 80px;
+}
+
+.new-posts:hover {
+    background: grey;
+}
```

Comment:
I'd probably move these classes to their own file — `application.css` can often grow out of control if it is added to often! Maybe `navigation.css`, or something similar.